src/spec/tests/static/defaults.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+describe("defaults: change default plugin options", function() {
+
+	var defaults = $.extend({}, $.fn.intlTelInput.defaults);
```

Comment:
Did you mean to use this somewhere? I guess in the afterEach, to reset the defaults again?